Libraries/DeprecatedPropTypes/DeprecatedLayoutPropTypes.js

```diff
@@ -0,0 +1,203 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict
+ */
+
+'use strict';
+
+const ReactPropTypes = require('prop-types');
+
+/**
```

Comment:
Can you move this comment to the file with Flow types as well? You can put it on top of `____LayoutStyle_Internal`

Reply:
I fixed 👍